### ISSUE TITLE
Grappelli autocomplete field in nested inline got initialized twice if parent inline has autocomplete field with same name

### ISSUE THREAD

**Issue by joseph2002** [theatlantic/django-nested-admin#57]
As described, consider the case below:

```
class ChildInline(NestedStackedInline):
    fields = ('name', 'category')
    raw_id_fields = ('category',)
    autocomplete_lookup_fields = {'fk': ('category',)}

class ParentInline(NestedStackedInline):
    fields = ('name', 'category')
    inlines = (ChildInline,)
    raw_id_fields = ('category',)
    autocomplete_lookup_fields = {'fk': ('category',)}
```

the autocomplete field 'category' in ChildInline will be initialized twice, so that two input box will appear.
in `DJNesting.initAutocompleteFields`,  `$('#' + prefix + '-group > .djn-items > *:not(.djn-empty-form)')
        .find('input[name^="' + prefix + '"][name$="-' + this + '"]')` will also find input in child inline with the same name.
currently, removing the 'category' field from autocomplete_loopup_fields in ChildInline seems ok.

**Comment by joseph2002** [comment]
also noticed that if I add more ChildInline item, the autocomplete filed in previous sibling ChildInline got init again

**Comment by fdintino** [comment]
This is fixed in the 3.0.12 release
